org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/bundlecontext/BundleContextSoftAssertions.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) OSGi Alliance (2019, 2020). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.osgi.test.assertj.bundlecontext;
+
+import org.assertj.core.api.SoftAssertionsProvider;
+import org.osgi.framework.BundleContext;
+
+public interface BundleContextSoftAssertions extends SoftAssertionsProvider {
```

Comment:
Shouldn't we name these soft assertion interfaces to end in `SoftAssertionsProvider` like the others? Seems like nice consistency.

Reply:
Absolutely!